docs/understanding-airbyte/airbyte-protocol.md

```diff
@@ -500,7 +503,6 @@ This table breaks down attributes of these state types.
 | Stream-Level Replication Isolation                   | X      |        |        |
 | Single state message describes full state for Source |        | X      | X      |
 
-- **Protocol Version** simply connotes which versions of the Protocol have support for these State types. The new state message is backwards compatible with the V1 message. This allows old versions of connectors and platforms to interact with the new message.
```

Comment:
This line was missing from the above graph. Decided removing was clearing than explaining.